app/views/admin/governments/index.html.erb

```diff
@@ -1,44 +1,47 @@
-<% page_title "Governments" %>
-<h1>Governments</h1>
+<% content_for :page_title, "Governments" %>
+<% content_for :title, "Governments" %>
 
 <% if can?(:manage, Government) %>
-  <p>
-    <%= link_to "Create a government", new_admin_government_path %>
-  </p>
+  <%= render "govuk_publishing_components/components/button", {
+    text: "Create a government",
+    href: new_admin_government_path,
+    secondary_solid: true,
+    margin_bottom: 6
+  } %>
 <% end %>
 
-<table class="governments table table-striped table-bordered add-top-margin">
-  <thead>
-    <tr class="table-header">
-      <th width="70%">Name</th>
-      <th width="15%">Start date</th>
-      <th width="15%">End date</th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @governments.each do |government| %>
-      <%= content_tag_for(:tr, government) do %>
-        <td>
-          <% if can?(:manage, Government) %>
-            <%= link_to government.name, edit_admin_government_path(government) %>
-          <% else %>
-            <%= government.name %>
-          <% end %>
-        </td>
-        <td>
-          <%= government.start_date.to_fs(:govuk_date) %>
-        </td>
-        <td>
-          <%= government.end_date.to_fs(:govuk_date) if government.end_date %>
-        </td>
-      <% end %>
-    <% end %>
+<% unless @governments.present? %>
+  <p class="govuk-body">No governments have been created.</p>
+<% end %>
 
-    <% if @governments.empty? %>
-      <tr>
-        <td colspan="3">No governments have been created.</td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<%= render "govuk_publishing_components/components/table", {
+  head: [
+    {
+      text: "Name"
+    },
+    {
+      text: "Start Date"
+    },
+    {
+      text: "End Date"
+    }
+  ],
+  rows:
+    @governments.map do |government|
+    [
+      {
+        text:  if can?(:manage, Government)
+                link_to(government.name, edit_admin_government_path(government))
```

Comment:
```
link_to(government.name, edit_admin_government_path(government), class: "govuk-link")
```

Reply:
+1 - all links in the design system should have `govuk-link` class